.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -32,6 +32,9 @@ If applicable, add screenshots and/or a video to help explain your problem.
 **Found in Branch**
 Name of or link to the branch where the issue occurs.
 
+**Commit ID from Branch**
+ID (SHA-1 hash) of the latest commit from the branch where the issue occurs.

```

Comment:
I would rephrase this for non-technical folks so it clear how to get the commit id and that commit ids only make sense if they are O3DE/O3DE commit ids.

Consider adding information about how to use `git rev-parse HEAD`or similar to get commit id

Reply:
For example, GoDot's bug report language is "Specify the Git commit hash if using a development or non-official build. If you use a custom build, please test if your issue is reproducible in official builds too."